File: perfs/java/Perf.java
import java.lang.Math; 
import java.util.Random;

public class Perf {
    private static Random random = new Random();

    public static long Sum(long a, long b){
        return a + b + random.nextInt();
    }

    public static void main(String args[]) {  
        long sum = 0;
        for(long count = 0; count < Math.pow(10, 8); count++)
        {
            sum = Sum(sum, count);
        }
    }
}

File: perfs/python/perf.py
import random
import math
import sys 

def Sum(a, b):
    return a + b + random.randint(0,1024)#sys.maxsize)

def main():
    sum = 0
    for count in range(0, 10**8):
        sum = Sum(sum, count)
    return sum

if __name__ == "__main__":
    main()